tests/test_setup_config_extended.py

```
@@ -297,6 +300,64 @@ def test_setup_downloads_both_selected(mocker, capsys):
+        config, is_partial_run=False, wants=lambda _: True
+    )
+
+    assert save_apks is True
+    assert save_firmware is False
+    assert result_config["CHECK_APP_SNAPSHOTS"] is True
+    assert result_config.get("APP_SNAPSHOT_VERSIONS_TO_KEEP", 1) >= 1
+
+
+@pytest.mark.configuration
+@pytest.mark.unit
+def test_setup_downloads_desktop_no_snapshot_prompt(mocker, capsys):
```

Ruff (0.15.20)

[warning] Unused function argument: `capsys`

(ARG001)